internal/ingress/controller/parser/kongstate/route.go

```diff
@@ -260,4 +289,18 @@ func (r *Route) overrideByKongIngress(log logrus.FieldLogger, kongIngress *confi
 	if ir.PathHandling != nil {
 		r.PathHandling = kong.String(*ir.PathHandling)
 	}
+	if len(ir.SNIs) != 0 {
+		var snis []*string
+		for _, sni := range ir.SNIs {
+			sanitizedSNI := strings.TrimSpace(*sni)
```

Comment:
nit: if you have two variables - one "safe" the other one "unsafe" (here: "sanitized" and "unsanitized", my preference is to annotate the "unsafe" one rather than the "safe one".

Here it would mean renaming:
sni -> either of inputSNI, unsanitizedSNI
sanitizedSni -> sni

Why? Defensive programming - a person editing this code in a year from now is somewhat likely to confuse the two; I have made this mistake dozens of times. It's better if the "safer" value has the "more appealing" name.

Another nit: acronyms should be either all caps or all small letters - this is standard in Go. Prefer `someAPI` to `someApi`, for example.